### PR TITLE
viewer - simplify code

### PR DESCRIPTION
**Pull request by severo**
It will be the pattern to use for #2068, since it allows to access the properties files only once for all the JSP files, and reduces the required code.

### CODE REVIEW

**Comment by severo** [comment]
Note that we could do the same for analytics, header, and other modules.

File: mapfishapp/src/main/webapp/WEB-INF/jsp/index.jsp [hunk 1]
```diff
@@ -65,7 +47,7 @@ ResourceBundle bundle = org.georchestra.mapfishapp.ws.Utf8ResourceBundle.getBund
 String detectedLanguage = rLocale.getLanguage();
 String forcedLang = request.getParameter("lang");
 
-String lang = defaultLanguage;
+String lang = request.getParameter("defaultLanguage");
```
Comment (fvanderbiest) [inline mapfishapp/src/main/webapp/WEB-INF/jsp/index.jsp]: There's no such parameter to my knowledge

Comment (severo) [reply mapfishapp/src/main/webapp/WEB-INF/jsp/index.jsp]: It's not the name of a property from a properties file, it's a parameter passed in the Spring configuration file, see https://github.com/georchestra/georchestra/pull/2430/files#diff-c15e9e41a281b1d28fad1be82868b16eR60

Comment (fvanderbiest) [reply mapfishapp/src/main/webapp/WEB-INF/jsp/index.jsp]: okay !

**Pull request merged**